mne/simulation/source.py

```diff
@@ -4,13 +4,99 @@
 #
 # License: BSD (3-clause)
 
+import os.path as op
+
 import numpy as np
 
+from ..label import grow_labels, Label
 from ..source_estimate import SourceEstimate, VolSourceEstimate
 from ..source_space import _ensure_src
+from ..surface import read_surface
 from ..utils import check_random_state, warn, _check_option
 
 
+def select_sources(subject, label, location='center', extent=0,
+                   grow_outside=True, subjects_dir=None, name=None,
+                   surf='white'):
+    """Select the sources used for simulation
+
+    This function selects a region of interest on the cortical surface based
+    on a label (or a hemisphere). The sources are selected by growing a region
+    around a seed which is selected randomly, is the center of the label, or
+    is a specific vertex. The selected vertices can extend beyond the initial
+    provided label. This can be prevented by setting grow_outside to False.
+
+    The selected sources are returned in the form of a new Label object. The
+    values of the label contain the distance from the seed in millimeters.
```

Comment:
We try to avoid long descriptions at the top unless necessary, I'd move both paragraphs to a new `Notes` section (also add `.. versionadded:: 0.18` directive to the bottom of the `Notes` section)